app/tests/components/Form/ApplicationForm.test.tsx

```diff
@@ -282,7 +282,7 @@ describe('The application form', () => {
 
     // luxon renders this differently in test?
     expect(
-      screen.getByText(/August 27, 2022, 9:51 a.m. PDT/)
+      screen.getByText(/August 27, 2022 at 9:51 a.m. PDT/)
```

Comment:
It's a slight change, but it does affect copy so making sure we confirmed this adjustment with PO?
```diff
- August 27, 2022, 9:51 a.m. PDT
+ August 27, 2022 at 9:51 a.m. PDT
```